examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala

```diff
@@ -70,15 +70,15 @@ object StructuredSessionization {
         line.split(" ").map(word => Event(sessionId = word, timestamp))
       }
 
-    // Sessionize the events. Track number of events, start and end timestamps of session, and
+    // Sessionize the events. Track number of events, start and end timestamps of session,
     // and report session updates.
     val sessionUpdates = events
       .groupByKey(event => event.sessionId)
       .mapGroupsWithState[SessionInfo, SessionUpdate](GroupStateTimeout.ProcessingTimeTimeout) {
 
         case (sessionId: String, events: Iterator[Event], state: GroupState[SessionInfo]) =>
 
-          // If timed out, then remove session and send final update
+          // If timed out, remove session and send final update
```

Comment:
Hm, other two occurrences are fine.

I'll just get this in - let's just keep the `and` typo fix only in this PR

```diff
-    // Sessionize the events. Track number of events, start and end timestamps of session, and
+    // Sessionize the events. Track number of events, start and end timestamps of session,
```

Reply:
> Hm, other two occurrences are fine.
> 
> I'll just get this in - let's just keep the `and` typo fix only in this PR
> 
> ```diff
> -    // Sessionize the events. Track number of events, start and end timestamps of session, and
> +    // Sessionize the events. Track number of events, start and end timestamps of session,
> ```

Is `Sessionize` a typo?